apps/wallet-mobile/src/TxHistory/ActionsBanner.tsx

```diff
@@ -200,25 +206,34 @@ const styles = StyleSheet.create({
   buyInfo: {
     fontSize: 16,
     color: '#000000',
-    fontFamily: 'Rubik',
+    fontFamily: 'Rubik-Regular',
     lineHeight: 24,
+    textAlignVertical: 'center',
+  },
+  bold: {
+    fontWeight: '500',
+    fontFamily: 'Rubik-Medium',
   },
   disabled: {
     opacity: 0.5,
   },
-  buyModalContent: {flex: 1, flexDirection: 'column', justifyContent: 'space-between', paddingBottom: 26},
+  buyModalContent: {flex: 1, flexDirection: 'column'},
 })
 
 const useStrings = () => {
   const intl = useIntl()
-  const bold = {b: (text) => <Text bold>{text}</Text>}
+
+  const formattingOptions = {
+    b: (text: ReactNode) => <Text style={[styles.buyInfo, styles.bold]}>{text}</Text>,
+    textComponent: (text: ReactNode) => <Text style={styles.buyInfo}>{text}</Text>,
+  }
```

Comment:
is it part of strings?

Reply:
imho, we should add styles only at the component, not in the strings, it looks odd to have bold here in the translations, is there a way to leave this decision up the component rendering the text?